src/components/policy/policy_external/src/policy_table/types.cc

```diff
@@ -736,6 +823,16 @@ void ModuleConfig::ReportErrors(rpc::ValidationReport* report__) const {
   if (!vehicle_year.is_valid()) {
     vehicle_year.ReportErrors(&report__->ReportSubobject("vehicle_year"));
   }
+#ifdef SDL_REMOTE_CONTROL
```

Comment:
if the consent is needed, please change the name from xxxPassenger to something more appropriate, otherwise please remove